packages/artifact/README.md

```diff
@@ -39,6 +39,10 @@ Method Name: `uploadArtifact`
     - If set to `false`, and an error is encountered, all other uploads will stop and any files that were queued will not be attempted to be uploaded. The partial artifact available will only include files up until the failure.
     - If set to `true` and an error is encountered, the failed file will be skipped and ignored and all other queued files will be attempted to be uploaded. There will be an artifact available for download at the end with everything excluding the file that failed to upload
     - Optional, defaults to `true` if not specified
+- `retentionDays`
+    - Durantion after which artifact will expire in days
+    - Minimum value: 1
+    - Maximum value: 90 unless changed by settings
```

Comment:
What will happen if the user puts in a negative value as input? Would it be worth it to have some input validation?

I would expand a little bit on the "changed by settings". Who can change these settings, and where can this be done? You put some more information in `upload-options.ts` and I think it would be good to put as much information in the `README` since users will look at this first.